release_notes/ocp-4-13-release-notes.adoc

```diff
@@ -120,6 +120,36 @@ As an administrator, you can specify multiple failure domains for your {product-
 
 [id="ocp-4-13-web-console"]
 === Web console
+[id="ocp-4-13-developer-perspective"]
+==== Developer Perspective
+
+With this release, there are several updates to the *Developer* perspective of the web console. You can perform the following actions:
+
+* Create a *Serverless Function* through the *Import from Git* flow.
+* Create a *Serverless Function* through *Create Serverless Function* flow available on *Add page*.
+* The *Import from Git* workflow now supports *pipeline-as-code*.
+* See which pods are receiving traffic in the:
+** Side pane of the *Topology* view
+** Pod *Details* view
+** *Pods* list view
+* Customize the timeout period or provide your own image when instantiating a *Web Terminal*.
+* Set default resources as an administrator to be pre-pinned in the *Developer* perspective navigation for all the users.
+
+[id="ocp-4-13-pipelines-page-improvements"]
+===== Pipelines page improvements
+In {product-title} 4.13, you can see the following navigation improvements on the *Pipelines* page:
+
+* The previously selected tab is visible when returning to the *Pipelines* page.
```

Comment:
```suggestion
* The tab you previously selected remains visible when you return to the *Pipelines* page.
```
- A couple of suggestions for improving user-centric wording

Reply:
Made this change. Thanks!